### PR TITLE
update release info for ukrainian story pages

### PR DESCRIPTION
**Pull request by rhenshaw56**
part of #7576 

**Overall change:**
_update release info for ukrainian story pages._

**Code changes:**
   - _Add asset paths to cypress/support/config/settings.js._
   - _Add local assets._
   - _Add STY launch dates to launchDates.js._

---

- [X] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [X] I have assigned this PR to the Simorgh project

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: cypress/support/config/settings.js [hunk 1]
```diff
@@ -7494,15 +7494,15 @@ module.exports = () => ({
...
             enabled: false,
           },
           test: {
-            paths: [],
+            paths: ['/ukrainian/23154182'],
             enabled: false,
           },
           local: {
-            paths: [],
+            paths: ['/ukrainian/23340963'],
```
Comment (karinathomasbbc) [inline cypress/support/config/settings.js:7505]: Are you also planning to set `enabled: true` here?

Comment (rhenshaw56) [reply cypress/support/config/settings.js:7505]: @karinathomas doing that ensures cypress tests run against that asset right ?

Comment (rhenshaw56) [reply cypress/support/config/settings.js:7505]: not sure if this is STY specific, but other page types seem to have this set to `true` except some STY's

Comment (karinathomasbbc) [reply cypress/support/config/settings.js:7505]: Yeah - probably a discussion to have with QA - would they like to have these URLs tests in E2Es on AWS?

Comment (rhenshaw56) [reply cypress/support/config/settings.js:7505]: most of those we have for STY's aren't enabled yet, is this something we'd like to turn on ? cc @joshcoventry @jimjohnsonrollings @andrewscfc

Comment (andrewscfc) [reply cypress/support/config/settings.js:7505]: Didn't we decide not to run E2Es for all page types as that took too long? I think we chose to do one LTR, one RTL and a few more for other services with significant differences, but I can't remember all of them.

**Review by andrewscfc — APPROVED** [review]

**Review by simonsinclair — APPROVED** [review]

**Pull request merged**